src/SprykerEco/Zed/Payone/Business/PayoneFacadeInterface.php

```diff
@@ -50,7 +55,7 @@ interface PayoneFacadeInterface
      *
      * @return void
      */
-    public function saveOrder(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponse);
+    public function saveOrder(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponse): void;
```

Comment:
It's a part of public API, please, remove added typehints.

Reply:
Fixed.